src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCache.java

```diff
@@ -90,6 +100,12 @@ public DefaultCache(
         this.cachedEntries = new Int2ObjectHashMap<>();
         this.counters = counters;
         this.supplyTrace = requireNonNull(supplyTrace);
+        totalSlots = cacheCapacity / cacheBufferPool.slotCapacity();
```

Comment:
Use `this.totalSlots` in constructor for consistency.

Reply:
Done.